src/app/app.component.html

```diff
@@ -1,34 +1,39 @@
-<div class="wrapper" [class.dark]="darkMode()">
+<div class="wrapper" [class.dark]="darkMode()" (click)="closeMobileMenu()">
   <header class="fixed w-full">
     <nav class="bg-white border-gray-200 py-2.5 dark:bg-gray-900">
       <div class="flex flex-wrap items-center justify-between max-w-screen-xl px-4 mx-auto">
         <a routerLink="/" class="flex items-center">
-          <img ngSrc="/assets/img/logo.png" class="h-6 mr-3 sm:h-9 w-auto" alt="{{'logo.alt' | transloco}}" height="455" width="393" priority/>
+          <img ngSrc="./assets/img/logo.png" class="h-6 mr-3 sm:h-9 w-auto" alt="{{'logo.alt' | transloco}}" height="455" width="393" priority/>
           <span class="self-center text-xl font-semibold whitespace-nowrap dark:text-white">{{'app_title' | transloco}}</span>
         </a>
         <div class="flex items-end lg:order-2">
           <button data-collapse-toggle="mobile-menu-2" type="button"
                   class="inline-flex items-center p-2 ml-1 text-sm text-gray-500 rounded-lg lg:hidden hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600"
-                  aria-controls="mobile-menu-2" aria-expanded="false">
+                  aria-controls="mobile-menu-2" aria-expanded="false"
+                  (click)="toggleMobileMenu(); $event.stopPropagation()"
+                  >
             <span class="sr-only">{{'open_main_menu' | transloco}}</span>
-            <inline-svg href="/assets/img/open-menu.svg" />
-            <inline-svg href="/assets/img/close-menu.svg" />
+            <inline-svg href="assets/img/open-menu.svg" />
+            <inline-svg href="assets/img/close-menu.svg" />
```

Comment:
Why this change? It's throughout the whole PR.

Reply:
The mobile experience hamburger menu did not seem to work, or if it did, it was in a non obvious way. This was my effort at making it work and work with a click away function to hide it

Reply:
Correct. We'll tackle this another time.

Reply:
Sorry, I misunderstood. This was so the site isn't bound to /, and would work on subdirs

Reply:
I meant the change that removes the leading `/` throughout the whole PR, sometimes replacing it with `./`

Reply:
Yes sorry, I realized after. I answered above

Reply:
Ah, nevermind, I haven't read the comment that came with the PR. Anyway, it would be better to either stick to `./` or no prefix, not mixing it up.